rclpy/rclpy/node.py

```diff
@@ -31,6 +31,19 @@
 from rclpy.validate_topic_name import validate_topic_name
 
 
+def check_for_type_support(msg_type):
+    try:
+        if msg_type.__class__._TYPE_SUPPORT is None:
+            msg_type.__class__.__import_type_support__()
+    except AttributeError:
+        raise AttributeError(
+            "unable to access '_TYPE_SUPPORT' attribute\n"
+            'Make sure that your ROS2 workspace s before your ROS1 workspace'
+            ' in your PYTHONPATH.')
```

Comment:
Maybe don't argue about the order in the environment variable but the order of sourcing (ROS 2 after ROS 1)?